tests/testthat/test-get_rd.R

```diff
@@ -12,8 +12,9 @@ test_that("Can find relevant R documentation files, given a .Rd filepath", {
 
 test_that("get_rd find macros properly", {
   rd_filepath <- system.file("examples", "rd_file_sample_2.Rd", package = "rd2markdown")
-  expect_s3_class(get_rd(file = rd_filepath, macros = NA), "Rd")
-  expect_silent(rd <- get_rd(file = rd_filepath, macros = NA))
+  rd_macro <- system.file("examples", "macros", "macros.Rd", package = "rd2markdown")
+  expect_s3_class(get_rd(file = rd_filepath, macros = rd_macro), "Rd")
+  expect_silent(rd <- get_rd(file = rd_filepath, macros = rd_macro))
```

Comment:
This test doesn't really test the discovery of package macros, which is a pretty central feature here. Can we also add a test for a macro we embed in this package? 

You can see a [trivial macro here in a package I was working on recently](https://github.com/openpharma/roxylint/blob/7-sentence-case/man/macros/nolint.Rd).

We could test it easily by using it in the package docs themselves or alternatively creating a tiny fixture package to test it against.

Reply:
Tests added!